frontend/packages/policy-editor/src/components/PolicyEditor.tsx

```diff
@@ -1,23 +1,25 @@
 import React, { useState } from 'react';
-import classes from './PolicyEditor.module.css';
-import { ExpandablePolicyCard } from 'resourceadm/components/PolicyEditor/ExpandablePolicyCard';
-import { CardButton } from 'resourceadm/components/PolicyEditor/CardButton';
 import { Label, Alert } from '@digdir/design-system-react';
 import {
+  PolicyActionType,
   PolicyBackendType,
-  PolicyRuleCardType,
   PolicyRuleBackendType,
+  PolicyRuleCardType,
+  PolicyRuleResourceType,
   PolicySubjectType,
   RequiredAuthLevelType,
-  PolicyActionType,
-} from 'resourceadm/types/global';
+  policyEditorUsageType,
+} from '@altinn/policy-editor/src/types';
 import {
   mapPolicyRulesBackendObjectToPolicyRuleCardType,
   emptyPolicyRule,
   mapPolicyRuleToPolicyRuleBackendObject,
-} from 'resourceadm/utils/policyEditorUtils';
-import { VerificationModal } from 'resourceadm/components/PolicyEditor/VerificationModal';
-import { SelectAuthLevel } from 'resourceadm/components/PolicyEditor/SelectAuthLevel';
+} from '@altinn/policy-editor/src/utils';
+import classes from './PolicyEditor.module.css';
+import { VerificationModal } from './VerificationModal';
+import { SelectAuthLevel } from './SelectAuthLevel';
+import { ExpandablePolicyCard } from './ExpandablePolicyCard';
+import { CardButton } from './CardButton';
 
 interface Props {
```

Comment:
What about using type instead of interface and renaming the `Props` to `PolicyEditorProps`? Then we are following the same naming convention as the rest of the repo.
```suggestion
type PolicyEditorProps {
```